test/unit/spec/PasswordReset_spec.js

```diff
@@ -54,6 +55,17 @@ function (Q, _, $, OktaAuth, LoginUtil, Util, PasswordResetForm, Beacon, Expect,
       delete settings.policyComplexity;
     }
 
+    if (settings && settings.policyAge) {
```

Comment:
this won't work if you have a test that asks for specific complexity and also policyAge
```
itp('has a valid subtitle if excludeUsername and password age "history" defined', function () {
    return setup({policyComplexity: 'excludeUsername', policyAge: 'history'}).then(function (test) {
```
which is the test I think would be missing.